src/Database/Driver.php

```diff
@@ -486,6 +519,16 @@ public function getMaxAliasLength(): ?int
         return static::MAX_ALIAS_LENGTH;
     }
 
+    /**
+     * Returns the number of connection retry attempts made.
+     *
+     * @return int
+     */
+    public function getConnectRetries(): int
```

Comment:
Would also need to add `@method` annotation to `DriverInterface` for this new method.